1.8/install/database-restore.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: "Database Restore"
+categories: [install]
+
+redirect_from:
+- /Database-Restore.html
+---
+
+To restore a database to your forum, you will first need to know what database engine your forum is using in order to determine which instructions to follow. MyBB supports the following database types:
+
+* MySQL (In this demonstration, PhpMyAdmin will be used)
```

Comment:
It might also be nice to link to the PhpMyAdmin/PhpLiteAdmin/PhpPgAdmin project homepages here I think.